components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/model/ActionInvocationResponse.java

```diff
@@ -65,6 +70,7 @@ public String getErrorLog() {
      */
     public enum Status {
         SUCCESS,
+        FAILURE,
```

Comment:
We discussed to have the failure status as 'FAILED' ?

Reply:
Addressed with commit 0812bd15f91ba1f366b024b26a5aa4e8178d752f